Airports/EGFH/Positions.txt

```diff
@@ -0,0 +1 @@
+EGFH_I_TWR:Swansea Radio:119.700:FHI:I:EGFH:TWR:-:-:::N051.36.19.000:W04.04.04.000::
```

Comment:
Since it's a "Radio" position, I think EGFH_**R**_TWR might be more appropriate, with identifier FH**R**.